client/webserver/site/src/html/markets.tmpl

```diff
@@ -449,17 +449,20 @@
             <span class="grey fs17 flex-grow-1 text-start">[[[Quantity]]]</span>
             <span id="vQty" class="fs18 demi"></span>
             <span data-unit="base" class="grey fs14 ms-2"></span>
+            <span id="vFiatQty" class="d-hide"> / &dollar; <span></span></span>
           </div>
           <div class="d-flex align-items-center mt-1">
             <span class="grey fs17 flex-grow-1 text-start">[[[Total]]]</span>
             <span id="vTotal" class="fs18 demi"></span>
             <span data-unit="quote" class="grey fs14 ms-2"></span>
+            <span id="vFiatTotal" class="d-hide"> / &dollar; <span></span></span>
```

Comment:
These two figures should be relatively equivalent. We only need to show the approximate fiat value once. How about a new row? Or just put the figure in small grey text at the bottom right of this section. And prefer "USD" to a dollar sign.

Reply:
> These two figures should be relatively equivalent. We only need to show the approximate fiat value once. How about a new row?

Does this mean we should only display fiat value for `Total`?